app/assets/stylesheets/application.css

```diff
@@ -121,3 +110,11 @@ a {
 .center {
   text-align: center;
 }
+
+.flex-col {
+  flex-direction: column;
+}
+
+.v-hidden {
+  visibility: hidden;
```

Comment:
- [ ] Good job adding devise to your project, however, you still need to style it to match the given wireframe as shown below. Please implement this feature

### Wireframe
![Screenshot 2023-09-08 at 14 20 57 (2)](https://github.com/DebyGrey/blog-app/assets/64774936/1f42ff81-8642-452e-b6da-0c5cc7b126bc)

### Your project
<img width="368" alt="Screenshot 2023-09-08 at 14 28 11" src="https://github.com/DebyGrey/blog-app/assets/64774936/bae516b2-b2d8-4729-afef-29613ae7b6f7">